src/Main.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by strejolopez on 11/29/16.
+
+public class Main {
```

Comment:
Before pushing lets get in the habit of removing the large blocks of commented out code.